NebulaPatcher/Patches/Transpilers/UIProductionStatWindow_Transpiler.cs

```diff
@@ -118,24 +118,30 @@ static IEnumerable<CodeInstruction> UpdatePower_Transpiler(IEnumerable<CodeInstr
                     {
                         codes[i - 2 + j].opcode = OpCodes.Nop;
                     }
-                    codes[i] = new CodeInstruction(OpCodes.Ldloca_S, 47); //Loads the address of the local variable at a specific index onto the evaluation stack, short form.
+                    codes[i] = new CodeInstruction(OpCodes.Ldloca_S, 42); //Loads the address of the local variable at a specific index onto the evaluation stack, short form.
                     codes[i + 1] = new CodeInstruction(OpCodes.Ldarg_0);
                     codes[i + 2] = new CodeInstruction(OpCodes.Ldfld, targetIndex);
                     codes[i + 3] = new CodeInstruction(OpCodes.Call, AccessTools.Method(typeof(StatisticsManager), "UpdateTotalChargedEnergy"));
                 }
             }
+
+            if(!patchActive[3])
```

Comment:
Again, debugging code - please either remove before merge or make more player-readable

Reply:
There are a few transpilers I want to rewrite, but I do want to keep some kind of check in here so we know when patches aren't working. How would you suggest I phrase the messages so that it is player-readable? I'm not the best with that stuff. As an example of how I think it should look:

https://github.com/Mushroom/nebula/blob/may2021-patch-fix/NebulaPatcher/Patches/Transpilers/BuildingParameters_Transpiler.cs#L41

But yeah I agree the wording isn't the best. Should it just be "X_Transpiler failed, mod version not compatible with game version." or something else?

Reply:
Perhaps something along the lines of: "Error in UpdatePower transpiler; patch index 3 inactive - additional info: "

Which then gives people clear additional info to provide us in the event of an error